From 79ab07db8d91b6c6e0c36358c8030893c28510ba Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Tue, 2 Aug 2016 10:32:53 -0400
Subject: [PATCH 1/1] clit.c: fix -Wformat-security warnings.

Two uses of printf() in clit.c were triggering -Wformat-security
warnings due to a missing "%s" format string. This was causing
compilation to fail with -Werror=format-security, so they have
been fixed.

Gentoo-Bug: 521246
---
 clit18/clit.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/clit18/clit.c b/clit18/clit.c
index c13a75d..48b749f 100644
--- a/clit18/clit.c
+++ b/clit18/clit.c
@@ -125,9 +125,9 @@ int main(int argc, char ** argv)
             dir_program[i+1] = '\0'; break;
         }
     }
-    printf(sTitle);
+    printf("%s", sTitle);
     if (argc < 3) { 
-        printf(sUsage);
+        printf("%s", sUsage);
         return -1;
     }
     base = 1;
-- 
2.7.3

